## [Copy files](https://symfony.com/doc/current/frontend/encore/copy-files.html)
Ok, I've pushed changed version for `copy-files`. 

For the other parts I have some issues:

## [Post CSS](https://symfony.com/doc/current/frontend/encore/postcss.html)
Skipping this step `yarn add postcss-loader autoprefixer --dev` results in the prompt that `postcss-loader` should be installed: `yarn add postcss-loader@^5.0.0 --dev`. That's ok, but when after adding `autoprefixer` in `postcss.config.js` file and running `yarn encore dev` I get an error:
```
Module Error (from ./node_modules/postcss-loader/dist/cjs.js):
Loading PostCSS "autoprefixer" plugin failed: Cannot find module 'autoprefixer'
```
What should be done in this case? Leaving only `yarn add autoprefixer --dev`?

## [ReactJS](https://symfony.com/doc/current/frontend/encore/reactjs.html)
Similar question. In the docs we are suggesting to install:
```
yarn add @babel/preset-react --dev
yarn add react react-dom prop-types
```

skipping this step results in a suggestion that only `yarn add @babel/preset-react@^7.0.0 --dev` should be installed. What about `react react-dom prop-types`?

--- a/frontend/encore/copy-files.rst
+++ b/frontend/encore/copy-files.rst
@@ -28,7 +28,7 @@ Referencing Image files from a Template
 
 To reference an image file from outside of a JavaScript file that's processed by
 Webpack - like a template - you can use the ``copyFiles()`` method to copy those
-files into your final output directory.
+files into your final output directory. First enable it in ``webpack.config.js``:
 
 .. code-block:: diff
 
@@ -50,6 +50,10 @@ files into your final output directory.
     +         // only copy files matching this pattern
     +         //pattern: /\.(png|jpg|jpeg)$/
     +     })
+
+Then restart Encore. When you do, it will give you a command you can run to
+install any missing dependencies. After running that command and restarting
+Encore, you're done!
 
 This will copy all files from ``assets/images`` into ``public/build`` (the output
 path). If you have :doc:`versioning enabled <versioning>`, the copied files will

--- a/frontend/encore/postcss.rst
+++ b/frontend/encore/postcss.rst
@@ -4,11 +4,26 @@ PostCSS and autoprefixing (postcss-loader)
 `PostCSS`_ is a CSS post-processing tool that can transform your CSS in a lot
 of cool ways, like `autoprefixing`_, `linting`_ and more!
 
-First, download ``postcss-loader`` and any plugins you want, like ``autoprefixer``:
+First enable it in ``webpack.config.js``:
+
+.. code-block:: diff
+
+      // webpack.config.js
+
+      Encore
+          // ...
+    +     .enablePostCssLoader()
+      ;
+
+Then restart Encore. When you do, it will give you a command you can run to
+install any missing dependencies. After running that command and restarting
+Encore, you're done!
+
+Next, download any plugins you want, like ``autoprefixer``:
 
 .. code-block:: terminal
 
-    $ yarn add postcss-loader autoprefixer --dev
+    $ yarn add autoprefixer --dev
 
 Next, create a ``postcss.config.js`` file at the root of your project:
 
@@ -23,19 +38,6 @@ Next, create a ``postcss.config.js`` file at the root of your project:
             autoprefixer: {}
         }
     }
-
-Then, enable the loader in Encore!
-
-.. code-block:: diff
-
-      // webpack.config.js
-
-      Encore
-          // ...
-    +     .enablePostCssLoader()
-      ;
-
-Because you just modified ``webpack.config.js``, stop and restart Encore.
 
 That's it! The ``postcss-loader`` will now be used for all CSS, Sass, etc files.
 You can also pass options to the `postcss-loader`_ by passing a callback:

--- a/frontend/encore/reactjs.rst
+++ b/frontend/encore/reactjs.rst
@@ -10,7 +10,6 @@ Using React? First add some dependencies with Yarn:
 
 .. code-block:: terminal
 
-    $ yarn add @babel/preset-react --dev
     $ yarn add react react-dom prop-types
 
 Enable react in your ``webpack.config.js``:

